### DISCUSSION TITLE
Allow reducing the space between column in the Sections layout type

### DISCUSSION THREAD

**Discussion by bleuthoot-sven** [home-assistant/frontend#20411] [Dashboard]
Compared to the Masonry layout, the new Sections layout has a lot of space between the columns. I would like an option to change the spacing between the cards.

This would help when displaying multiple columns on a single, small display. Currently I am using a Google Nest Hub as a display for my Home Assistant dashboard. With the Masonry view type, I get 3 columns of space. With the new Sections view type, I only get 2 columns of space.

Below are some simulated examples of my desired result using the inspect element tool with the display resolution of the Google Nest Hub

<details>
<summary>Masonry layout on a Google Nest Hub</summary>

![example-masonry ](https://github.com/home-assistant/frontend/assets/25549100/a9c1d9d0-2b94-4ce7-a9be-d23aad1dae0c)

</details>

<details>
<summary>Sections layout on a Google Nest Hub</summary>

![example-sections](https://github.com/home-assistant/frontend/assets/25549100/4be0a122-0cc8-409c-982c-754b1e0100d5)

</details>

<details>
<summary>Desired result</summary>

![example-sections-adjusted](https://github.com/home-assistant/frontend/assets/25549100/14f963e8-37ba-48b2-9995-463643038468)

</details>

**Comment by noxhirsch** [comment]
This theme variable should help: `ha-view-sections-column-gap: 8px`
Here are all the available theme variables for sections: https://github.com/home-assistant/frontend/pull/20099

**Reply by bleuthoot-sven** [reply]
Thanks, added this option to a theme and seems to look fine now. Seems like a good alternative until something like this is implemented.

**Reply by noxhirsch** [reply]
I assume this *is* the implementation. I don't think there will be an UI setting for this.

**Comment by vbtrek** [comment]
Where do you put this variable to customise the spacing on a sections view?

**Reply by bleuthoot-sven** [reply]
You have to add it to a theme. Easiest way is to create a new theme in the themes folder and add the following.

```yaml
Reduced spacing theme:
     ha-view-sections-column-gap: 8px
```

**Reply by vbtrek** [reply]
Awesome, got it thanks.

**Comment by Stooovie** [comment]
"Add it to a theme" as in, edit  the themes yaml? Because that doesn't do anything even after reseting the UI cache. I use Noctis or Metro.

**Reply by gio-dot** [reply]
Do HA system complete reboot or use service reload_theme. It must work

**Comment by TestNamePleaseIgnoreMe** [comment]
Any way to increase the columns on the Google Nest Hub from 2 columns to 3?
